### PR TITLE
Junits for References and Lost/Found, small fixes

### PR DESCRIPTION
**Pull request by claudiasima-ro**
Change-Id: Ib71476a38ea951c1c2c132771207c645fa30874b
Signed-off-by: claudiasima <claudia.sima@thalesgroup.com>

### CODE REVIEW

File: tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/ReferencesXtextToDiagramTestCase.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.polarsys.capella.scenario.editor.ju.testcases;
```
Comment (georgiana-ecobici) [inline tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/ReferencesXtextToDiagramTestCase.java:13]: Add Thales headers

File: tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/LostAndFoundXtextToDiagramTestCase.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package org.polarsys.capella.scenario.editor.ju.testcases;
```
Comment (georgiana-ecobici) [inline tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/LostAndFoundXtextToDiagramTestCase.java:13]: Add Thales headers.

File: tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/ReferencesXtextToDiagramTestCase.java (hunk 1 above)
Comment (claudiasima-ro) [reply tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/ReferencesXtextToDiagramTestCase.java:13]: Done

File: tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/LostAndFoundXtextToDiagramTestCase.java (hunk 2 above)
Comment (claudiasima-ro) [reply tests/org.polarsys.capella.scenario.editor.ju/src/org/polarsys/capella/scenario/editor/ju/testcases/LostAndFoundXtextToDiagramTestCase.java:13]: Done

**Review by georgiana-ecobici — APPROVED** [review]

**Pull request merged**